src/utils/URLUtils.test.ts

```diff
@@ -20,12 +20,6 @@ import {
 
 describe('URLUtils', () => {
   describe('cortexScorecardPageURL', () => {
-    it('should use default cortex url', () => {
-      expect(cortexScorecardPageURL({ scorecardId: '1' })).toEqual(
```

Comment:
impossible to call this without specifying a url now so no default!

Reply:
maybe we can add a test somewhere else to test the actual hook using the default correctly?

Reply:
Definitely testable (https://react-hooks-testing-library.com/) but I don't consider it a blocker for this.

Reply:
and backstage kindly provided `TestApiProvider` so we can mock the config api